lms/envs/common.py

```diff
@@ -2118,14 +2118,16 @@
 
 # JWT Settings
 JWT_AUTH = {
-    'JWT_SECRET_KEY': None,
+    # TODO Set JWT_SECRET_KEY to a secure value. By default, SECRET_KEY will be used.
+    # 'JWT_SECRET_KEY': '',
```

Comment:
Is devOps aware of this change?  The security implications are probably fine.  But it does introduce a coupling between the keys - so if/when we want to change one, we will break compatibility with other things.  This may be perfectly fine, but just an FYI.